addons/mass_mailing_sms/controllers/main.py

```diff
@@ -89,10 +89,10 @@ def blacklist_number(self, mailing_id, trace_code, **post):
             'unsubscribe_error': unsubscribe_error,
         })
 
-    @http.route('/r/<string:code>/s/<int:sms_sms_id>', type='http', auth="public")
-    def sms_short_link_redirect(self, code, sms_sms_id, **post):
-        if sms_sms_id:
-            trace_id = request.env['mailing.trace'].sudo().search([('sms_sms_id_int', '=', int(sms_sms_id))]).id
+    @http.route('/r/<string:code>/s/<int:sms_sms_id_int>', type='http', auth="public")
+    def sms_short_link_redirect(self, code, sms_sms_id_int, **post):
```

Comment:
Note the old naming was correct, we effectively have a SMS ID ^^ does not change anything, could reduce diff :D

Reply:
helps when grepping